stdlib/3/venv/__init__.pyi

```diff
@@ -33,9 +35,14 @@ class EnvBuilder:
     def post_setup(self, context: SimpleNamespace) -> None: ...
     def replace_variables(self, text: str, context: SimpleNamespace) -> str: ...  # undocumented
     def install_scripts(self, context: SimpleNamespace, path: str) -> None: ...
+    if sys.version_info >= (3, 9):
+        def upgrade_dependencies(context: SimpleNamespace) -> None: ...
```

Comment:
The type for `context` could be improved with a Protocol but that's out of scope for this diff.